### DISCUSSION TITLE
サイトマップのupdatedの日付がpublishedと同じになってしまう

### DISCUSSION THREAD

**Discussion by massan00** [miyadaiku/miyadaiku#42] [Q&A] [answered]
サイトマップで出力されるファイルのpublishedとupdatedが同じ日付になってしまいます。
記事にはdate,updatedともに設定しています。

サイトマップのupdatedの日付を記事のupdatedの日付と合わせるにはどうしたらいいでしょうか。
サイトマップはatom.xmlです。

**Comment by atsuoishimoto** [comment, marked as answer]
気がついてませんでしたが、feedgeneratorは日付を一つだけ指定するとpublishedとupdatedに反映してくれるんですね

1.21.0で、ファイルにupdatedを指定しておくと、feedのupdatedとして出力するように修正しました。`pip install -U miyadaiku` として、試してみて下さい

**Reply by massan00** [reply]
さっそく使わせていただきました。updated,publishedともに反映されました。

ありがとうございます。
